core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -1080,4 +1083,39 @@ class ControllerApis(
         requestThrottleMs => new AssignReplicasToDirsResponse(reply.setThrottleTimeMs(requestThrottleMs)))
     }
   }
+
+  def handleAddRaftVoter(request: RequestChannel.Request): CompletableFuture[Unit] = {
+    authHelper.authorizeClusterOperation(request, ALTER)
+    val addRequest = request.body[AddRaftVoterRequest]
+    raftManager.handleAddVoter(addRequest.data())
+    requestHelper.sendResponseMaybeThrottle(request, requestThrottleMs =>
+      new AddRaftVoterResponse(new AddRaftVoterResponseData().
+        setThrottleTimeMs(requestThrottleMs).
+        setErrorCode(0.toShort).
+        setErrorMessage(null)))
+    CompletableFuture.completedFuture[Unit](())
```

Comment:
I think this is going to change in the real implementation. For KRaft requests we simply forward them to the raft thread/actor using `handleRaftRequest` and `raftManager.handleRequest`. 

How about for now we just send an error response with "unsupported version" and return a completed future?

This comment applies to all of the new methods added to this type.

Reply:
ok